inginious/frontend/i18n/es/LC_MESSAGES/messages.po

```diff
@@ -3287,6 +3547,10 @@ msgstr ""
 "Este es tu grupo inicial. El grupo no se puede eliminar y tampoco los "
 "estudiantes."
 
+#: inginious/frontend/templates/course_admin/classroom_edit.html:112
+msgid "Classroom description"
+msgstr "Descripción de grupo"
```

Comment:
```suggestion
msgstr "Descripción del salón"
```

Reply:
This little change was not made because another spanish translations on this context were similar.

Reply:
Ok, No problem